modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleContainer.tsx

```diff
@@ -107,6 +109,14 @@ export default function ScheduleContainer({
 			displayTitle={Liferay.Language.get('schedule')}
 			displayType="secondary"
 		>
+			<div className="mb-3 text-justify">
```

Comment:
Hi @larissacribeiro, could we add these styles in the css file?

Reply:
sure, thanks!